api/schemas/report.json

```diff
@@ -135,14 +134,6 @@
             "type":        "string"
         },
 
-        "kind": {
```

Comment:
Awesome. Well, count this as a heads up if something ends up breaking... it doesn't make sense to have anymore, given we've reduced it to one option.